R/rhtmlLabeledScatter.R

```diff
@@ -36,6 +36,7 @@
 #' @param labels.font.color is the font color of the labels. NOTE: This overrides the color if it is set
 #' @param labels.font.size is the font size of the labels
 #' @param labels.logo.scale is a vector of scaling factors for label logos
+#' @param labels.max.shown Number of labels to show on chart.
```

Comment:
Maybe explain a bit more how this works, e.g. which labels are hidden and mention how toggling overrides this